kong/tools/utils.lua

```diff
@@ -1468,4 +1468,82 @@ do
 end
 _M.time_ns = time_ns
 
+
+local sha256
```

Comment:
I think the name should be `sha256_bin`, just like ngx_lua's `md5_bin`

Reply:
i think that was a mistake by ngx.